core/db/curibio/alembic/versions/da5a93c1bb08_update_uploadtype_and_scope_schema.py

```diff
@@ -0,0 +1,50 @@
+"""update UploadType and scope schema
+
+Revision ID: da5a93c1bb08
+Revises: 4aac7b465d29
+Create Date: 2022-10-26 07:20:30.894705
+
+"""
+from alembic import op
+import json
+
+# revision identifiers, used by Alembic.
+revision = "da5a93c1bb08"
+down_revision = "4aac7b465d29"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # default all existing customers and users to paid pulse3d
+    op.execute(f"update users set data='{json.dumps({'scope': ['pulse3d:user:paid']})}'")
+    op.execute(f"update customers set data='{json.dumps({'scope': ['pulse3d:customer:paid']})}'")
+
+    # change mantarray uploadtype to pulse3d
+    # set existing mantarray types to other accepted type that is not currently being used
+    op.execute("""UPDATE uploads SET type='nautilus' WHERE type='mantarray'""")
```

Comment:
Do these need to be in triple quotes?

Reply:
Nah, just noticed all the other migrations had it too, i'll remove them